tools/pull_request_hooks/removeGuideComments.js

```diff
@@ -18,6 +18,12 @@ function escapeRegex(string) {
 export async function removeGuideComments({ github, context }) {
   let newBody = context.payload.pull_request.body;
 
+  // sometimes we make PRs without PR bodies and the whole thing will fail
```

Comment:
Comment shouldn't be necessary, the code is obvious

Reply:
cruft from earlier implementation